tools/localizations/cn.csv

```diff
@@ -96,3 +96,4 @@ Aqara Wireless Mini Switch T1,Aqara 无线开关 T1
 "Zemismart M6 Hub",Zemismart M6 网关
 "Siterwell Smart Smoke Alarm",赛特威尔智能烟雾报警器
 "Siterwell Motion Sensor",赛特威尔人体检测传感器
+"Greentown Lock(SG20)",绿城门锁(SG20)
```

Comment:
This string needs to match the label used in the fingerprint in order to be translated, so either this string needs to change to `Greentown Door Lock` or the `deviceLabel` needs to be changed to `Greentown Lock(SG20)`